docs/user/manage_users.rst

```diff
@@ -0,0 +1,107 @@
+
+.. _manage_users:
+
+Manage Users
+~~~~~~~~~~~~
+
+You can search for, filter, add, and edit user accounts in Kolibri from the **Users** tab in your **Manage** dashboard.
+
+.. image:: img/manage_users.png
+  :alt: manage users
+
+.. note::
+  To manage Kolibri users you must be logged-in as **Device Owner** or **Admin**.
+
+
+Kolibri User Roles
+------------------
+
+Kolibri users can have different roles with respective access to features:
+
+* **Learners** can:
+  
+  * View content and have their progress tracked
+* **Coaches** can:
+  
+  * View content and have their progress tracked
+  * View *Coach Reports* to track progress of other users and usage stats for individual exercises
+  * Create/Edit/Delete *Classes* and *Groups* and enroll users in them
```

Comment:
Coach users cannot create, edit, or delete classes. They can only do this with groups